src/tools/perf/perftest.c

```diff
@@ -241,6 +241,27 @@ static void print_progress(char **test_names, unsigned num_names,
     fflush(stdout);
 }
 
+static char *overhead_lat(struct perftest_context *ctx)
```

Comment:
@zhuyj maybe define the name of the column inside test_type_t ? so for every test case the name will be either "overhead" or "latency"